index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portofolio - Here we go</title>
+  </head>
+  <body>
+    <div id="app">
+      <header class="header">
+        <nav class="menu">
+          <a href="#"><h2>My Logo</h2></a>
+        </nav>
```

Comment:
- [ ] As pointed out by the previous reviewer, please let's remove the `<nav></nav> ` element around the `<a></a> ` element.

Reply:
Wonderful

Reply:
But I said to wrap <a></a> element with <nav></nav>  in the previous comment and here you say again that I need to remove It.  What's the difference?

Reply:
But you said in the second comment that I need to wrap the <a></a> with <nav></nav> and in this last comment you ask me to remove them again. PLease tell me, what is the difference?

Reply:
https://www.w3schools.com/tags/tag_nav.asp, this is one of my reference.